tool_quality_checks/test/test_db.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Tests for db.py"""
+import os
+import sys
+import pytest
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
Ik heb BHV nodig...

Reply:
Haha. Ik lag in een deuk. Gelukkig heb je zelf BHV ;).

Zie verder bovenstaande opmerking